engine/swarm/admin_guide.md

```diff
@@ -251,8 +237,68 @@ Before you forcefully remove a manager node, you must first demote it to the
 worker role. Make sure that you always have an odd number of manager nodes if
 you demote or remove a manager
 
+## Back up the swarm
+
+Docker manager nodes store the swarm state and manager logs in the
+`/var/lib/docker/swarm/raft` directory. In 1.13 and higher, the Raft data is
+encrypted, and the encryption keys are stored in
+`/var/lib/docker/swarm/certificates`. Without these keys, you will not be able
+to restore the swarm.
+
+You can back up the swarm using any manager. Use the following procedure.
+
+1.  Stop Docker on the manager before backing up the data, so that no data is
+    being changed during the backup. It is possible to take a backup while the
+    manager is running (a "hot" backup), but this is not recommended and your
+    results will be less predictable when restoring. While the manager is down,
+    other nodes will continue generating swarm data that will not be part of
+    this backup.
+
+    > **Note**: Be sure to maintain the quorum of swarm managers. For instance,
+    > if you normally run with three managers, you need to add a manager before
+    > you can stop one, because two managers are not sufficient to maintain the
```

Comment:
Two managers are a quorum given a cluster of 3 managers, so you shouldn't need to add another before bringing this one down.  Maybe something like "...if you normally run with 3 managers, and one of them is unreachable, stopping one of the remaining ones will break quorum and disable the cluster until either the unreachable one or the stopped one comes back up"?